cumulus/polkadot-parachain/src/main.rs

```diff
@@ -14,11 +14,66 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Cumulus test parachain collator
+//! Polkadot parachain-omni-node.
 
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+pub(crate) const EXAMPLES: &str = color_print::cstr!(
+	r#"<bold><underline>Examples:</></>
+
+   <bold>polkadot-parachain-omni-node --chain para.json --sync warp -- --chain relay.json --sync warp</>
+        Launch a warp-syncing full node of a given para's chain-spec, and a given relay's chain-spec.
+
+	<green><italic>The above approach is the most flexible, and the most forward-compatible used to spawn an omni-node.</></>
+
+	You can find the chain-spec of some networks in:
+
+	https://paritytech.github.io/chainspecs
+
+   <bold>polkadot-parachain-omni-node --chain asset-hub-polkadot --sync warp -- --chain polkadot --sync warp</>
```

Comment:
The meat of the omni node is knowing how to use the `--chain`. But this flag is coming from the depth of the substrate's CLI types. I wish there was a way to customize the doc you get from `--chain` to, for example, demonstrate the list of options for you.

Reply:
I'm not sure if this is possible. I will check.

Reply:
I couldn't find a way to do this so far